sdk/monitor/monitor-query/README.md

```diff
@@ -394,6 +395,99 @@ async function processTables(tablesFromResult: LogsTable[]) {
 
 A full sample can be found [here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-query/samples/v1/typescript/src/logsQueryBatch.ts).
 
+### Resource Logs Query
+
+The following example demonstrates how to query logs directly from an Azure resource without the use of a Log Analytics workspace. Here, the query_resource method is used instead of query_workspace, and instead of a workspace ID, an Azure resource identifier is passed in (e.g. /subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/{resource-provider}/{resource-type}/{resource-name}).
```

Comment:
```suggestion
The following example demonstrates how to query logs directly from an Azure resource without the use of a Log Analytics workspace. Here, the `queryResource` method is used instead of `queryWorkspace`, and instead of a workspace ID, an Azure resource ID is passed in (e.g. /subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/{resource-provider}/{resource-type}/{resource-name}).
```